chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
```

Comment:
Proper comment not given 
suggestion 
before defining the class/function as what it does